core/src/modules/ops/destination.ts

```diff
@@ -40,53 +44,79 @@ export namespace DestinationOps {
   /**
    * Export all the Group Members of the Groups that this Destination is Tracking
    */
-  export async function exportGroupMembers(
-    destination: Destination,
-    force = false
-  ) {
-    const group = await destination.$get("group");
-    if (group) return group.run(force, destination.id);
+  export async function exportMembers(destination: Destination, force = false) {
+    if (destination.collection === "group") {
+      const group = await destination.$get("group");
+      if (group) return group.run(force, destination.id);
+    } else if (destination.collection === "model") {
+      const model = await destination.$get("model");
+      if (model) return model.run(force, destination.id);
+    } else {
+      throw new Error(`cannot export members for a ${destination.collection}`);
+    }
   }
 
-  /**
-   * Track a Group
-   */
-  export async function trackGroup(
+  export async function updateTracking(
     destination: Destination,
-    group: Group,
-    force = true
+    collection: Destination["collection"] | typeof NullKey,
```

Comment:
Oh, I see `NullKey` is able to be passed for the `collection`, but I don't think it's being handled?